app/views/content/welcome/my-journey-into-teaching/_three-things-youll-never-hear-me-say.html.erb

```diff
@@ -1,44 +1,27 @@
-<section class="story three-things-youll-never-hear-me-say">
-  <%= image_pack_tag "media/images/content/hero-images/0011.jpg" %>
+<% if subject_specific_story_data[:three_things_youll_never_hear_me_say].present? %>
+  <section class="case-study three-things-youll-never-hear-me-say">
+    <%= image_pack_tag "media/images/content/hero-images/0011.jpg" %>
 
-  <h2>
-    <div class="caption">
-      <span class="purple">
-        Three things you'll
-      </span>
-    </div>
-    <div class="caption blend">
-      <span class="purple">
-        <em>never</em> hear me say.
-      </span>
-    </div>
-  </h2>
+    <h2>
+      <div class="caption">
+        <span class="purple">
+          Three things you'll
+        </span>
+      </div>
+      <div class="caption blend">
+        <span class="purple">
+          <em>never</em> hear me say.
```

Comment:
Personally I don't like using all caps for NEVER - I don't think is good in terms of accessibility

Is this a standard style or can we remove the spans and just have it as 'never'?

Reply:
Ok, I'll remove the `text-tranform: uppercase` for the moment. I agree about the underline, but I'll leave that in and we can address the pattern more widely and consistently.